tests/utilities/test_fetching.py

```diff
@@ -11,16 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from time import time
+from typing import Any
+
 import pytest
+import torch
 from torch import tensor
-from torch.utils.data import DataLoader, IterableDataset
+from torch.utils.data import DataLoader, Dataset, IterableDataset
 
+from pytorch_lightning import Trainer
 from pytorch_lightning.trainer.supporters import CombinedLoader
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from pytorch_lightning.utilities.fetching import DataFetcher
+from pytorch_lightning.utilities.fetching import DataFetcher, DataLoaderIterDataFetcher, InterBatchParallelDataFetcher
+from tests.helpers.boring_model import BoringModel
+from tests.helpers.runif import RunIf
 
 
-@pytest.mark.parametrize("use_combined_loader", [False, True])
+@pytest.mark.parametrize("use_combined_loader", [False])
```

Comment:
Is this change on purpose or leftover debugging?

Reply:
This is done on purpose. Unittesting with CombinedLoader too.